### ISSUE TITLE
Using your code

### ISSUE THREAD

**Issue by mattleon09** [mvarnagiris/financius#410]
I just wanted to let you know that I have started working with your code. You can contact me at mattleondev@gmail.com

**Comment by mvarnagiris** [comment]
Are you going to push PRs here?

**Comment by mattleon09** [comment]
No, I will be sending PRs to my git. Making sure that you are credited properly for your code of course.

**Comment by marcin-kozinski** [comment]
@mvarnagiris If we send PRs here, for example to fix some of the other open issues, are you going to merge them and release an update to the Play Store?
